### PR TITLE
Sorter

### PR DESCRIPTION
**Pull request by cyklon73**

### CODE REVIEW

File: src/main/java/de/cyklon/reflection/function/Sorter.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import de.cyklon.reflection.types.Loadable;
+import de.cyklon.reflection.types.Modifiable;
+import de.cyklon.reflection.types.Nameable;
+
+import java.util.Comparator;
+
+public interface Sorter<T> extends Comparator<T> {
+
+	static <T> Sorter<T> random() {
+		return (o1, o2) -> Math.round((float) (Math.random()*3-2));
```
Comment (MineKing9534) [inline src/main/java/de/cyklon/reflection/function/Sorter.java:12]: Why?

File: src/main/java/de/cyklon/reflection/types/Modifiable.java [hunk 2]
```diff
@@ -5,7 +5,13 @@
 import java.util.EnumSet;
 
 public interface Modifiable {
-	@NotNull EnumSet<Modifier> getModifiers();
+
+	int getIntModifiers();
```
Comment (MineKing9534) [inline src/main/java/de/cyklon/reflection/types/Modifiable.java:9]: Rename to getEncodedModifers

**Review by MineKing9534 — CHANGES_REQUESTED** [review]

File: src/main/java/de/cyklon/reflection/function/Sorter.java (hunk 1 above)
Comment (cyklon73) [reply src/main/java/de/cyklon/reflection/function/Sorter.java:12]: At the moment there is no real benefit, but in some cases you have to shuffle a list or something.

Comment (MineKing9534) [reply src/main/java/de/cyklon/reflection/function/Sorter.java:12]: I still don't see the purpose. Why should this library provide a random sorter? This library will not need this for sure and if someone needs this they should implement it them self. This is out of scope for this lib.

**Review by MineKing9534 — APPROVED** [review]

**Pull request merged**